specification/cognitiveservices/AnomalyDetector/multivariate/models.tsp

```diff
@@ -25,19 +25,19 @@ enum MultivariateBatchDetectionStatus {
 }
 
 @doc("Data schema of the input data source. The default is OneTable.")
-enum DataSchema {
-  @doc("OneTable means that your input data is in one CSV file, which contains one time stamp column and several variable columns. The default DataSchema value is OneTable.")
+enum MultivariateDataSchema {
+  @doc("OneTable means that your input data is in one CSV file, which contains one time stamp column and several variable columns. The default MultivariateDataSchema value is OneTable.")
   "OneTable",
-  @doc("MultiTable means that your input data is separated in multiple CSV files. Each file contains one time stamp column and one variable column, and the CSV file name should indicate the name of the variable. The default DataSchema value is OneTable.")
+  @doc("MultiTable means that your input data is separated in multiple CSV files. Each file contains one time stamp column and one variable column, and the CSV file name should indicate the name of the variable. The default MultivariateDataSchema value is OneTable.")
   "MultiTable",
 }
 
-enum AlignMode {
+enum MultivariateAlignMode {
   "Inner",
   "Outer",
 }
 
-enum ModelStatus {
+enum MultivariateModelStatus {
```

Comment:
Ask Ted about this name change.

Reply:
@tg-msft Could you help share your opinions on this?

Reply:
Mike and I had a quick chat about this and I agree that it's a little odd to change many of the `Model*` names to `MutlivariateModel*` but leave `ModelDetails`.  I think it makes sense to change that one too.

Additionally, Mike made the good point that the only remaining difference is then `AnomalyDetectionModel` and it would probably be cleaner to change that to `MultivariateModel` so it both aligns with all the other names and also leaves you more room to grow your API in the future (i.e., if you ever add a new class of model in future releases, you won't have customers getting confused why the best named type `AnomalyDetectionModel` doesn't work with the big new feature area, etc.).

Reply:
Fixed.